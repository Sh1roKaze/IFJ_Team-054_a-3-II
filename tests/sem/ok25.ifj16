class Main
{
	static void run()
	{
		int a = ifj16.readInt();
		double b = ifj16.readDouble();
		String c = ifj16.readString();
		ifj16.print(a + " " + b + " " + c + "\n");
		ifj16.print(1);
		ifj16.print(1.1);
		ifj16.print("a");
		ifj16.print(1+1.1);
		ifj16.print("aa" + 1 + "\n");
		a = ifj16.length("aa");
		c = ifj16.substr(c, a, 5);
		a = ifj16.compare("aaa",c);
		ifj16.print(a);
		a = ifj16.find(c,"s");
		ifj16.print(a);
		c = ifj16.sort(c);
		ifj16.print(c);
	}
}
