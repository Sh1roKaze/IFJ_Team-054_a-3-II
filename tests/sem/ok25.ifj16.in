4
2.7E+1
Ahojstring
