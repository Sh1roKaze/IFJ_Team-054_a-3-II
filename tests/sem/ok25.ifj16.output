4 27 Ahojstring
11.1a2.1aa1
-12jorst
