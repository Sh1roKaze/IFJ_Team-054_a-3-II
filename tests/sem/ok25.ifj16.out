4 27 Ahojstring
11.1a11.1aa1
-12jorst